media-gfx/gnome-photos/gnome-photos-43.0.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit gnome.org gnome2-utils meson python-any-r1 virtualx xdg
+
+DESCRIPTION="Access, organize and share your photos on GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Photos"
+
+LICENSE="GPL-3+ LGPL-2+ CC0-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="test upnp-av"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=x11-libs/cairo-1.14.0
+	>=x11-libs/gdk-pixbuf-2.36.8:2
+	>=media-libs/gegl-0.4.0:0.4[cairo,raw]
+	>=sci-geosciences/geocode-glib-3.26.4-r200:2
+	>=media-libs/gexiv2-0.14.0
+	>=dev-libs/glib-2.62.0:2
```

Comment:
Looks like some new dependencies were added right here:

````diff
+libportal_dep = dependency('libportal')
+libportal_gtk3_dep = dependency('libportal-gtk3')
````

Needs to depend on `dev-libs/libportal[gtk]`.

Reply:
Isn't libportal a flatpak thing? We have -Dflatpak=false